pkg/asset/ignition/bootstrap/vsphere/template.go

```diff
@@ -0,0 +1,20 @@
+package vsphere
```

Comment:
similar comment to https://github.com/openshift/installer/pull/3268/files#r390624093